modules/common/service/service.go

```diff
@@ -105,6 +151,16 @@ func (s *Service) CreateOrPatch(
 		h.GetLogger().Info(fmt.Sprintf("Service %s - %s", service.Name, op))
 	}
 
+	// update the service instance with the ip/host information
+	s.clusterIP = service.Spec.ClusterIP
+	s.serviceHostname = fmt.Sprintf("%s.%s.svc", service.Name, service.GetNamespace())
+
+	if service.Spec.Type == corev1.ServiceTypeLoadBalancer {
+		if len(service.Status.LoadBalancer.Ingress) > 0 {
+			s.externalIP = service.Status.LoadBalancer.Ingress[0].IP
```

Comment:
can this have multiple IPs?

Reply:
I guess in case of dualstack https://metallb.universe.tf/usage/#ipv6-and-dual-stack-services

Reply:
probably should make this externalPs for this case